src/Containers/Microsoft.NET.Build.Containers/Layer.cs

```diff
@@ -186,9 +187,17 @@ static UnixFileMode DetermineFileMode(FileSystemInfo file)
         string contentHash = Convert.ToHexString(hash).ToLowerInvariant();
         string uncompressedContentHash = Convert.ToHexString(uncompressedHash).ToLowerInvariant();
 
+        string layerMediaType = manifestMediaType switch
+        {
+             // TODO: configurable? gzip always?
+            SchemaTypes.DockerManifestV2 => "application/vnd.docker.image.rootfs.diff.tar.gzip",
+            SchemaTypes.OciManifestV1 => "application/vnd.oci.image.layer.v1.tar+gzip",
```

Comment:
Would you mind extracting these to constants like we've slowly been doing for the other media types?